src/pages/Bed/BedBuilder/BedBuilder.js

```diff
@@ -1,30 +1,32 @@
 import React, { useCallback, useState } from "react"
+import PropTypes from "prop-types"
+
 import BedMeasure from "./BedMeasure"
 import BedGrid from "./BedGrid"
 import "./BedBuilder.css"
 
 const MAX_WIDTH = 20,
-  MAX_HEIGHT = 20,
-  DEF_WIDTH = 0,
-  DEF_HEIGHT = 0
+  MAX_HEIGHT = 20
 
-export const BedBuilder = () => {
+const BedBuilder = ({ height, width, unit }) => {
   const [measurements, setMeasurements] = useState({
-    width: DEF_WIDTH,
-    height: DEF_HEIGHT,
-    unit: "cm",
+    width,
+    height,
+    unit,
   })
 
   const handleBedSettings = useCallback((measurement) => {
     setMeasurements(measurement)
   })
 
+  console.log(measurements.unit)
```

Comment:
Might wanna clean this up if you don't do so in another commit

Reply:
Heard! I'll do it in my next commit.